ext/intl/bidi/bidi.c

```diff
@@ -330,32 +353,84 @@ static PHP_METHOD(IntlBidi, setPara) {
 }
 /* }}} */
 
-/* {{{ proto self IntlBidi::setLine(int $tart, int $limit, IntlBidi $line) */
-ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(bidi_setline_arginfo, ZEND_RETURN_VALUE, 3, IS_OBJECT, 0)
+/* {{{ proto IntlBidi IntlBidi::setLine(int $start, int $limit) */
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(bidi_setline_arginfo, ZEND_RETURN_VALUE, 2, IS_OBJECT, 0)
 	ZEND_ARG_TYPE_INFO(0, start, IS_LONG, 0)
 	ZEND_ARG_TYPE_INFO(0, limit, IS_LONG, 0)
-	ZEND_ARG_TYPE_INFO(0, line, IS_OBJECT, 0)
 ZEND_END_ARG_INFO();
 static PHP_METHOD(IntlBidi, setLine) {
 	zend_long start, limit;
-	zend_object *zline;
-	php_intl_bidi_object *objval = bidi_object_from_zend_object(Z_OBJ_P(getThis()));
-	php_intl_bidi_object *lineval;
+	zval retval;
+	zend_function * constructor;
+	zend_fcall_info fci;
+	zend_fcall_info_cache fcc;
+	php_intl_bidi_object *objval, *lineval;
 	UErrorCode error;
 
-	if (zend_parse_parameters_throw(ZEND_NUM_ARGS(), "llO", &start, &limit, &zline, php_intl_bidi_ce) == FAILURE) {
-		return;
+	// Parse parameters.
```

Comment:
The comments in here feel superfluous.  Keep them if you want, but they're all rather... self-referential.

Reply:
Yeah, most of them where superflous, i removed most of them. (I left the TODO for now).